chart/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
Let's add a bit to this doc explaining that if it's running on http, there will need to be a TLS ingress in front of it. (We could also mention that you can use localhost and proxy, but that's such a hack that I don't want to).

And refer the user back to examples of using cert-manager and Google managed certs I guess.

Reply:
I will add this all to the docs

Reply:
Damn still need to do this